iib/workers/config.py

```diff
@@ -132,6 +136,7 @@ class TestingConfig(DevelopmentConfig):
     iib_greenwave_url = 'some_url'
     iib_omps_url = 'some_url'
     iib_request_logs_dir = None
+    iib_request_related_bundles_dir = None
```

Comment:
Why TestingConfig rewrites `iib_request_related_bundles_dir` from DevelopmentConfig?

Reply:
DevelopmentConfig is used by the dev env and so we need the value there. For TestingConfig, since it inherits the values, they will be set. But we actually use a separate `tmpdir` as the location for those files for tests. We do not want to unnecessarily populate that location by the tests that might be written wrong. Hence the overwriting.

Reply:
Sounds good. It removes ambiguity in the tests.